src/libraries/System.IO.Compression.ZipFile/tests/ZipFile.Unix.cs

```diff
@@ -0,0 +1,114 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+namespace System.IO.Compression.Tests
+{
+    public class ZipFile_Unix : ZipFileTestBase
```

Comment:
Do we have or need any tests that validate we're correctly _not_ setting the external attributes on Windows?

Reply:
Not yet. I'll add one.

Reply:
Added one. I even found some existing compat .zip files in our TestData that have the ExternalAttributes already set - looks like they were created on Linux and macOS given their names. So I also added a new test for them as well.